### DISCUSSION TITLE
Nornir inside Airflow-docker

### DISCUSSION THREAD

**Discussion by resparis** [nornir-automation/nornir#791] [General]
Hi, I am running a nornir (netmiko) script in airflow-docker to ssh into the devices. When the dag is trigger I got the exception: 

> paramiko.ssh_exception.SSHException: No authentication methods available

Anyone knows a way to deal with ssh keys in docker?

**Comment by ktbyers** [comment]
You probably need to downgrade to Paramiko 2.8.1. Can you try this and see if it works?
